configs/_base_/datasets/semantic_kitti.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+
+# dataset settings
+dataset_type = 'SemanticKittiDataset'
+data_root = 'data/semantickitti/'
+class_names = [
+    'unlabeled', 'car', 'bicycle', 'motorcycle', 'truck', 'bus', 'person',
+    'bicyclist', 'motorcyclist', 'road', 'parking', 'sidewalk', 'other-ground',
+    'building', 'fence', 'vegetation', 'trunck', 'terrian', 'pole',
+    'traffic-sign'
+]
+
+point_cloud_range = [0, -40, -3, 70.4, 40, 1]
+input_modality = dict(use_lidar=True, use_camera=False)
+
+db_sampler = dict(
+    data_root=data_root,
+    info_path=data_root + 'semantickitti_infos.pkl',
+    rate=1.0,
+    prepare=dict(
+        filter_by_difficulty=[-1],
+        filter_by_min_points=dict(Car=5, Pedestrian=10, Cyclist=10)),
+    classes=class_names,
+    sample_groups=dict(Car=12, Pedestrian=6, Cyclist=6))
+
+file_client_args = dict(backend='disk')
+# Uncomment the following if use ceph or other file clients.
+# See https://mmcv.readthedocs.io/en/latest/api.html#mmcv.fileio.FileClient
+# for more details.
+# file_client_args = dict(
+#     backend='petrel', path_mapping=dict(data='s3://kitti_data/'))
+
+train_pipeline = [
+    dict(
+        type='LoadPointsFromFile',
+        coord_type='LIDAR',
+        shift_height=True,
+        load_dim=4,
+        use_dim=[0, 1, 2]),
+    dict(
+        type='LoadAnnotations3D',
+        with_bbox_3d=False,
+        with_label_3d=False,
+        with_mask_3d=False,
+        with_seg_3d=True,
+        seg_3d_dtype=np.int32),
```

Comment:
This type can be set by default when loading annotations. It is a little inappropriate to set it here.

Reply:
That has been deleted in the new version.

Reply:
The setting about "LoadAnnotations3D" has been deleted.